packages/aws-lambda-otel-extension/opt/otel-extension/internal/index.js

```diff
@@ -329,6 +329,28 @@ const instrumentations = [
         eventData[context.awsRequestId].eventCustomRequestTimeEpoch =
           event.requestContext.timeEpoch;
       }
+
+      // Send request data to external so that we can attach this data to logs
+      await fetch(`http://localhost:${process.env.MOCK_PORT || OTEL_SERVER_PORT}`, {
+        method: 'post',
+        body: JSON.stringify([
+          {
+            time: new Date().toISOString(),
+            type: 'function',
+            record: {
+              recordType: 'eventData',
+              eventData,
+              span: {
+                traceId: span.spanContext().traceId,
+                spanId: span.spanContext().spanId,
+              },
+            },
+          },
+        ]),
```

Comment:
Ok, thanks for clarification!

Reply:
But why then we use AWS Lambda API log-like request format. 

It's a bit confusing

Reply:
I could change the format to something else if we would like I just though it would make the log filtering easier. Do you have a preference on the format?

Reply:
What information is important in that payload? Wouldn't payload as `{ eventData, traceId, spanId }` be good enough?

And if it's one of the different requests for this custom server, I'd probably use different paths for it (e.g. `/event-data/` in this case)

Reply:
> Why exactly easier, do we mix these requests, with requests that are coming from AWS logs API? (If so, I would definitely use different processors for them, so it's not that spaghetti)

Yeah we are using the same http server code for the Lambda Logs API and the custom HTTP server for telemetry data.

I can split these two http servers out but if you wouldn't mind I would prefer to do that work in a separate PR as I think it will be somewhat involved.

Reply:
> How important is it to have this PR in shortly?
The other console team mates were hoping to get their hands on this today so they could start testing infra as quickly as possible. I guess if we want to make that refactor first I can just start working on the other PR now.

Reply:
Actually I would also prefer to merge this in first so I don't have to refactor both the existing code in main and the code in this PR. If I merge this in then I could just refactor on codebase.

Reply:
Ok, no problem 👍

Reply:
Im going to resolve this conversation since we ended up doing this in this PR 👍